lib/checks/report/report.rb

```diff
@@ -0,0 +1,28 @@
+module Checks
+  module Report
+    class Report
```

Comment:
Reckon you're better off moving report.rb into lib and dropping this slightly odd namespacing.